src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -393,16 +393,8 @@ final class OnboardingInfoViewController: UIViewController {
 		case .privacyPage:
 			persistTimestamp(completion: completion)
 		case .enableLoggingOfContactsPage:
-			func handleBluetooth(completion: @escaping () -> Void) {
```

Comment:
Sorry but I don't get it. As I understood the Issue is that in the Dev and Community Builds, but not in the Production Build(!) a Screen is shown twice during onboarding. How is this fixing this Issue? Feels like the root cause lays somewhere else. Also it seems dangerous to just remove a presentation of an Alert during the onboarding.

Reply:
ok , I got it as a review from @whiskey  , the problem is that there is two calls for completion on in alertForBluetoothOff(completion:  and a second one in handleBluetooth(completion:  itself  so when u push a ViewControlelr  it's pushed twice  @marcussc

Reply:
The issue only happens in Debug Mode - so it's not happening in Release build. I'm not sure if removing `handleBluetooth` might be be correct solution here.

Reply:
I added it again in https://github.com/corona-warn-app/cwa-app-ios/pull/2217/commits/1d6663efe470ccab75a0ad6e005682230d425392 @KaiTeuber

Reply:
I just wonder why build and test checks not working for me @KaiTeuber

Reply:
@mozead1996 This is the wrong approach. There was a faulty call of the completion handler in the `MockExposureManager`. And since this is used in the simulator environment it resulted in the bug. Check #2225 for the fix.